roles/bwc_repos/tasks/bwc_repos_apt.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: Assert that master_token is specified
+  fail: msg="License key must be supplied for BWC enterprise installation."
+  when: license is not defined
+
+- name: Install prereqs (Debian)
+  become: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - debian-archive-keyring
+    - apt-transport-https
+
+- name: Get GPG key URL for private repo
+  become: yes
+  shell: 'curl "{{ deb_gpg_key_url }}"'
+  register: deb_gpg_key
+
+# This is the exact key as the open source repo but this behavior might change. So just
+# following what's in packagecloud docs for private repos.
+- name: Add GPG key to keyring
+  become: yes
+  shell: 'curl -L "{{ deb_gpg_key.stdout }}" | apt-key add -'
+  when: ansible_os_family == "Debian" and license is defined
```

Comment:
We never reach this task if license is not defined, because of `fail` task in this playbook.